src/test/java/org/apache/commons/text/similarity/FuzzyScoreTest.java

```diff
@@ -28,48 +27,49 @@
  */
 public class FuzzyScoreTest {
 
-    private static FuzzyScore score;
-
-    @BeforeClass
-    public static void setUp() {
-        score = new FuzzyScore();
-    }
-
     @Test
     public void testGetFuzzyScore() throws Exception {
-        assertEquals(0, (int) score.compare("", "", Locale.ENGLISH));
+        FuzzyScore score = new FuzzyScore(Locale.ENGLISH);
```

Comment:
Even simpler:

    private static final ENGLISH_SCORE = new FuzzyScore(Locale.ENGLISH);

Is that OK, or are there reasons to use ```@BeforeClass``` (more JUnit-y) or ```@Before``` (not static)?

Thanks.